src/icon/wheelchair.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import cc from 'classcat'
+import isEmpty from 'lodash.isempty'
+
+import { color } from '_utils/branding'
+
+class WheelchairIcon extends PureComponent<Icon> {
+  static defaultProps: Icon = {
```

Comment:
Out of scope for this PR but it hurts to see so much duplicated code in the Icon code :/

Reply:
Indeed...

Reply:
#221